Mathlib/Data/Nat/Fib/Basic.lean

```diff
@@ -83,6 +83,20 @@ lemma fib_add_one : ∀ {n}, n ≠ 0 → fib (n + 1) = fib (n - 1) + fib n
 
 theorem fib_le_fib_succ {n : ℕ} : fib n ≤ fib (n + 1) := by cases n <;> simp [fib_add_two]
 
+lemma fib_le_fib_of_le {m n : ℕ} (hmn : m ≤ n) :
```

Comment:
This already exists as [`Nat.fib_mono`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Nat/Fib/Basic.html#Nat.fib_mono).